build/commands/lib/config.js

```diff
@@ -397,26 +397,19 @@ Config.prototype.buildArgs = function () {
   versionParts = versionParts.split('.')
 
   let args = {
+    'import("//brave/build/brave_defaults.gni")': null,
```

Comment:
let's create a separate dir for these files. maybe `build/args`.

Reply:
Done.